source/documentation/monitoring_apps/metrics.md

```diff
@@ -1,28 +1,60 @@
-
 # Monitoring apps
 
 ## Metrics
 
-Cloud Foundry provides time-series data, or metrics, for each instance of your PaaS app. You should use the metrics exporter app to send this data to a monitoring system of your choice. Your monitoring system can then store this data over time and let you view it.
+Cloud Foundry provides time-series data, or metrics, for each instance of your PaaS app. You can receive, store and view this data over time in your chosen monitoring system by either:
+
+- using the [Prometheus](https://prometheus.io/) [external link] endpoint provided by the GOV.UK PaaS team
+- deploying the `paas-metric-exporter` app to push metrics data in [StatsD](https://github.com/etsy/statsd/wiki) [external link] format
+
+You can also view all metrics as a one-off snapshot by installing the Cloud Foundry CLI [log cache plug-in](https://github.com/cloudfoundry/log-cache-cli#installing-plugin). 
+
+### Prometheus
+
+The GOV.UK PaaS team maintains the API that Prometheus uses for free, which ensures that you can access all available metrics. You can configure Prometheus manually to filter out any unwanted metrics.
+
+You must set up Prometheus to request metrics from the `https://metrics.cloud.service.gov.uk/metrics` API endpoint.
+
+1. [Install Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/) [external link].
 
->You can also view data as a one-off snapshot using the Cloud Foundry CLI.
+1. You must set up a bearer token so the API endpoint can authenticate your Prometheus request. We recommend that you use a `bearer_token_file` as it is easy to maintain. Set up an automated cron job to run the following command every 5 minutes:
 
-To use the metrics exporter, you deploy it as an app on PaaS. The current metrics supported by the metrics exporter app are CPU, RAM, disk usage data, app crashes, app requests and app response times.
+	```
+	cf oauth-token > BEARER_TOKEN_FILE
```

Comment:
This is in capitals here and lower case everywhere else.

Reply:
I think it needs to differ to every where else, as it suppose to represent `/path/to/bearer_token_file.txt` - perhaps changing to that?

Reply:
To me that goes without saying. If they are referring to the same file they should be consistent.

Reply:
When referring to a parameter or something similar in code, the usual content pattern is to put that parameter in capitals and say "where `PARAMETER CODE` is the xyz." I've deviated from that slightly here because I think the text flows well. Happy to discuss.

Reply:
I could change it to:

```
cf oauth-token > BEARER_TOKEN_FILE
```
where:
- `cf oauth-token` is the command that generates a bearer token
- `BEARER_TOKEN_FILE` is the location and name of the`bearer_token_file` used by the Prometheus configuration

I think this is clear and should also fulfil accessibility requirements for screenreaders etc

Reply:
That seems exactly the same to me: one thing written in two different ways. Maybe it is just me that finds that inconsistency confusing and unnecessary. I don't really have a strong opinion on it though, so just leave it.